### PR TITLE
chore(northlight): new labelSize prop for SwitchField

### PR DESCRIPTION
**Pull request by ilija-**
- labelSize added
- Flex / Alignment fixes

closes: DEV-8557

### CODE REVIEW

File: framework/lib/components/switch/switch-field.tsx [hunk 1]
```diff
@@ -4,39 +4,73 @@ import { SwitchFieldProps } from './types'
...
  */
+
 export const SwitchField = ({
   name,
   label,
   isRequired,
   validate,
   onChange: onChangeCallback = identity,
   direction = 'row',
+  labelPosition = 'right',
```
Comment (unganemo) [inline framework/lib/components/switch/switch-field.tsx:40]: maybe change to labelPlacement to have more continuity? (iconPlacement in FlipButton)

File: framework/lib/components/switch/switch-field.tsx [hunk 2]
```diff
@@ -4,39 +4,73 @@ import { SwitchFieldProps } from './types'
...
+            name={ name }
+            onChange={ (e) => {
+              onChange(e)
+              onChangeCallback(e)
+            } }
+            value={ value }
+            data-testid="switch-field-test-id"
+            { ...rest }
+          />
+          { label && <Label htmlFor={ name } sx={ { fontWeight: 'normal' } } size={ labelSize }>{ label }</Label> }
```
[the remark this answers is not in the document]
Comment (ilija-) [reply framework/lib/components/switch/switch-field.tsx:72]: it is not.

**Comment by TautvydasGec** [comment]
Squash all commits

File: framework/lib/components/switch/switch-field.tsx [hunk 3]
```diff
@@ -58,7 +69,7 @@ export const SwitchField = ({
             data-testid="switch-field-test-id"
             { ...rest }
           />
-          { label && <Label htmlFor={ name }>{ label }</Label> }
+          { <Label htmlFor={ name } sx={ { fontWeight: 'normal' } } size={ labelSize }>{ label }</Label> }
```
Comment (unganemo) [inline framework/lib/components/switch/switch-field.tsx:72]: remove curly braces here, that's what's making the test fail

**Review by unganemo — CHANGES_REQUESTED** [review]

**Comment by TautvydasGec** [comment]
To squash all commits do this:
1. `git config --global core.editor "code --wait"` (only need this once) next time you do this you dont need this part
2. go to master and do `git pull`
3. go to your branch and do `git rebase -i master`
4. VS code window will open up. click on top right corner to enable oldest first
5. on the first line select pick and on all the other lines squash
6. cmd+Enter
7. write your commit message (This will have all your commit messages smushed together in one file)
8. exit window.
9. `git push --force`
10. profit

**Review by unganemo — CHANGES_REQUESTED** [review]
All flip button changes should not be in this PR. Two weeks late but hey better late than never

**Comment by ilija-** [comment]
> All flip button changes should not be in this PR. Two weeks late but hey better late than never

@unganemo 

This happened when I squashed the commit along with your commits, somehow.
There is also an update on the CheckboxField that you might be interested in, which is already merged. The same new props as this one.

**Comment by orestisrodriguez** [comment]
@unganemo @ilija- this seems against the composability that northlight and chakra has. the icon imo should be a children and not part of the flip button itself, and you can pass whatever directional property to it?

```
<FlipButtonGroup>
  <FlipButton flexDirection="row">
    <Icon ... />
    <Text>...</Text>
  </FlipButton>
</FlipButtonGroup>
```

**Review by unganemo — APPROVED** [review]

**Pull request merged**